files/mods.json

```diff
@@ -889,7 +888,7 @@
         "creator": "Cow",
         "discordcode": "fU2tFPf",
         "command": "/moo config",
-        "description": "A collection of various features and enhancements mainly towards Hypixel Skyblock with other features like log searching, stalk, \nbest friends list which are designed to be suitable for all Hypixel players.",
+        "description": "Collection of (Skyblock) features and tweaks like log searching, stalk, and a best friends list",
```

Comment:
why the ()?

Reply:
to preserve the "mainly towards Hypixel Skyblock"
should i remove it?

Reply:
i think removing the () and adding "miscellaneous" or something before "tweaks" would be better? i feel like currently its confusing for a user who has no clue what collection is